Assets/new fun logo

jbpenrath opened the pull request:
## Purpose

On FUN Mooc catalog we replaced the FUN Mooc logo by the new FUN logo. We want to do the same for the FUN Mooc LMS.

We also take opportunity to replace the old OpenEdX logo by the new one.

### Header
<img width="1000" alt="Before" src="https://user-images.githubusercontent.com/9265241/229100858-e2df6a87-237e-4b39-8dc0-03189c5c2446.png">
<img width="1000" alt="After" src="https://user-images.githubusercontent.com/9265241/229100302-51717417-b0e3-4ebc-82c1-e9a4e720a101.png">



### Footer
<img width="1000" alt="Before" src="https://user-images.githubusercontent.com/9265241/229100992-5778478d-c1f8-4128-8cc1-158b5ef57205.png">


<img width="1000" alt="After" src="https://user-images.githubusercontent.com/9265241/229100314-52df18fa-f8d7-4b8a-9b06-32ed292f981b.png">

jbpenrath committed:
🍱(funsite) add new fun logos

We want to replace FUN Mooc logos by the new France Université logos.

jbpenrath committed:
💄(funsite) use new fun logos

Update template and styles to use the new FUN logos in menu and footer.

jbpenrath committed:
💄(funsite) use new OpenEdX logo

We take the opportunity to replace FUN Mooc to the new FUN Logo to replace also the old OpenEdX logo by the new one.

sampaccoud reviewed (approved):
I think there is a "-" too many in one of the commit messages. 
Apart from that LGTM!

jbpenrath committed:
🔖(minor) bump version to 5.14.0

Changed
- Replace FUN Mooc logo with new FUN logo into menu and footer
- Use the new OpenEdX logo in the footer
